Cargo.toml

```diff
@@ -100,5 +100,5 @@ url = "2.4"
 uuid = { version = "1.6", features = ["v4"] }
 webpki-roots = "0.22"
 x509-cert = { version = "0.2", features = ["pem"] }
-x509-parser = { version = "0.15.1", features = ["verify"] }
+x509-parser = { version = "0.16", features = ["verify"] }
```

Comment:
can you also update `x509-parser` in `kmip` crate (dev dependencies) to unify versions? Workspace dep can be used there by the way